js/apps.js

```diff
@@ -0,0 +1,78 @@
+function App(name, logo, href) {
+    this.title = name;
+    this.logo = logo;
+    this.href = href;
+}
+
+var DOMAIN = "https://channeli.in/";
+
+buysell = new App("Buy & Sell", "buysell", "buyandsell");
+downloadsoftware = new App("Download Software", "downloadsoftware", "softwares");
+//examsonline = new App("Exams Online", "examsonline", "");
+//grades = new App("Grades", "grades", "");
+kriti = new App("Kriti", "kriti", "kriti");
+lectut = new App("LecTut", "lectut", "lectut");
+lostfound = new App("Lost & Found", "lostfound", "lostfound");
+messmenu = new App("Mess Menu", "messmenu", "messmenu");
+noticeboard = new App("Notice Board", "noticeboard", "notices");
+placement = new App("Placement Online", "placement", "placement");
+pplsearch = new App("People Search", "pplsearch", "peoplesearch");
+//questionairre = new App("Questionairre", "questionairre", "");
+//regol = new App("Regol", "regol", "");
+research = new App("Research Assistant", "research", "research_assistant");
+shp = new App("Student Home Page", "shp", "SHP");
+thinktank = new App("Thinktank", "thinktank", "thinktank");
+//vle = new App("VLE", "vle", "");
+
+var apps = [
+    buysell,
+    downloadsoftware,
+    //examsonline,
+    //grades,
+    kriti,
+    lectut,
+    lostfound,
+    messmenu,
+    noticeboard,
+    placement,
+    pplsearch,
+    //questionairre,
+    //regol,
+    research,
+    shp,
+    thinktank
+    //vle
+];
```

Comment:
why not this ?

```js
var app = {
  buysell: new App("Buy & Sell", "buysell", "buyandsell"),
  // others
}
```

Accessing this will be also easy by iteration through the keys.